doc/wui/ReleaseNotes.md

```diff
@@ -0,0 +1,7 @@
+Web User Interface V1
```

Comment:
All the files in our repo use snake_case naming convention. Please rename this one to `release_notes.md` 🙃

Reply:
we have not agreed on any naming convention yet

Reply:
@joshymjose please rename to release_notes.md

Reply:
there are many other files not following his rule eg README.md